android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -1070,4 +1076,18 @@ protected void disableHWAcceleration()
 	{
 		Log.d(TAG, "Do not disable HW acceleration for WebView.", Log.DEBUG_MODE);
 	}
+
+	public void setSoftwareMode(Boolean value)
+	{
+		WebView webView = getWebView();
+		if (webView != null) {
+			softwareMode = value;
```

Comment:
Is there a reason to save it as a local variable?

Reply:
I use that in the getter to return it